docs/arc42/12_glossary.md

```diff
@@ -0,0 +1,6 @@
+# Glossary
+
+| Term    | Definition                                                                                                                                                                                                                                                       |
+|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| AA      | amino acid                                                                                                                                                                                                                                                       |
+| aligned | a nucleotide sequence is aligned, if it is arranged such that it has many similarities to a given reference genome. The aligned sequence has the same length as the reference genome. Gaps are marked in the aligned sequence. Insertions are stored separately. |
```

Comment:
Add:

- QC: quality control; in our case, it usually refers to the quality checks and metrics of the sequences
- Variant: We follow a very open definition of variants. Every subset of sequences is considered a variant. A variant is specified by lineage/clade names and mutations. A variant does not need to be [monophyletic](https://en.wikipedia.org/wiki/Monophyly).